lib/l10n/intl_es.arb

```
@@ -743,5 +743,92 @@
   "@_comment_timeout_messages": "Mensajes de notificación de timeout",
   "orderTimeoutTaker": "No respondiste a tiempo. La orden será republicada",
   "orderTimeoutMaker": "Tu contraparte no respondió a tiempo. La orden será republicada",
-  "orderTimeout": "Tiempo de espera de la orden agotado"
+  "orderTimeout": "Tiempo de espera de la orden agotado",
+  
+  "@_comment_notifications": "Texto de Pantalla de Notificaciones",
+  "notifications": "Notificaciones",
+  "noNotifications": "Sin notificaciones",
+  "noNotificationsDescription": "Aún no tienes notificaciones. Las nuevas alertas de intercambio y actualizaciones aparecerán aquí.",
+  "errorLoadingNotifications": "Error al cargar notificaciones",
+  "retry": "Reintentar",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)